hws/src/dr_common_functions.py

```diff
@@ -137,20 +137,22 @@ def dr_parse_add_action(action_dw_0, action_dw_1, parse_value=True):
 
 def dr_parse_remove_by_size_action(action_dw_0, action_dw_1):
     action = {"type" : "Remove by size"}
-    start_anchor = int(action_dw_0[10 : 16], 2)
+    cx8 = True if _config_args.get("cx8") else False
+    start_anchor = int(action_dw_0[8 : 15], 2) if cx8 else int(action_dw_0[10 : 16], 2)
     field = modify_pattern_anchor_dic.get(start_anchor)
     action["start_anchor"] = field if field != None else start_anchor
     action["outer_l4_removed"] = int(action_dw_0[16 : 17], 2)
-    action["start_offset"] = int(action_dw_0[18 : 25], 2)
+    action["start_offset"] = int(action_dw_0[15 : 23], 2) if cx8 else int(action_dw_0[18 : 25], 2)
     action["size"] = int(action_dw_0[26 : 32], 2)
 
     return action
 
 
 def dr_parse_remove_header2header_action(action_dw_0, action_dw_1):
     action = {"type" : "Remove header2header"}
-    start_anchor = int(action_dw_0[10 : 16], 2)
-    end_anchor = int(action_dw_0[18 : 24], 2)
+    cx8 = True if _config_args.get("cx8") else False
```

Comment:
so total diff is
4 actions?
dr_parse_remove_by_size_action
dr_parse_remove_header2header_action
dr_parse_insert_inline_action
dr_parse_insert_by_pointer_action

future, if we will have one more device and more diffs these if`s will grow,
you will need a profile/config per device,

Reply:
yes
if need more changes in the future will change the whole logic